content/docs/replication/_index.md

```diff
@@ -19,13 +19,20 @@ CSM for Replication provides the following capabilities:
 | Capability                                                                          | PowerMax | PowerStore | PowerScale | PowerFlex | Unity |
 | ----------------------------------------------------------------------------------- | :------: | :--------: | :--------: | :-------: | :---: |
 | Replicate data using native storage array based replication                         | yes      | yes        | yes        | no        | no    |
+| Asynchronous file volume replication                         | yes      | yes        | yes        | no        | no    |
```

Comment:
To be confirmed by the engineering team, I think we do async block (yes) and not async file (no).

Reply:
Ah right, I updated the spreadsheet but not here.  Will update this.

Reply:
this should be updated with latest commit